readme.md

```diff
@@ -36,6 +36,12 @@ console.log(cleanStack(error.stack));
 Error: Missing unicorn
     at Object.<anonymous> (/Users/sindresorhus/dev/clean-stack/unicorn.js:2:15)
 */
+
+// If the stack is undefined or not a string, it returns undefined.
+const errorWithUndefinedStack = new Error();
+delete errorWithUndefinedStack.stack;
+
+console.log(cleanStack(error.stack)) // undefined
```

Comment:
I don't see the point of an example. This should just be a sentence in the `stack` docs (in the TS type docs too).